Automations - first roundtrip for forecasts (#2290)
FlexMeasures/flexmeasures: documentation/cli/commands.rst

``flexmeasures add annotation``                   Add annotation to accounts, assets and/or sensors.
``flexmeasures add toy-account``                  Create a toy account, for tutorials and trying things.
``flexmeasures add report``                       Create a report.

> ``flexmeasures add automation``                   Add a forecast automation with its own cron timezone.

Author: ```suggestion
``flexmeasures add automation``                   Add an automation with its own cron timezone.
```

``flexmeasures jobs run-job``                     Run a single job (useful for debugging it)
``flexmeasures jobs inspect-job``                 Inspect a background job and print its current status, result and metadata.
``flexmeasures jobs stats``                       Show estimated live statistics of the queueing system.

> ``flexmeasures jobs run-automations``             Handle due and missed forecast automation occurrences (run once per minute, e.g. via cron).

Author: I suggest replacing "occurrence" with "run" everywhere..
